### PR TITLE
enhancement: convert all png to webP

### LINKED ISSUE #5854: Large Image Sizes and PNG Format Causing Performance Issues.
**Current Behavior**
The current images used in the project are of heavy size, and many are in PNG format. This is causing performance issues, especially with page loading times. To improve performance, we need to:

**Desired Behavior**

Convert the PNG images to the WebP format, which offers better compression and smaller file sizes.
Reduce the size of other heavy images without compromising too much on quality.
Tasks:

- [ ]  Identify all PNG images in the project.
- [ ]  Convert PNG images to WebP format.
- [ ]  Reduce the size of large images while maintaining acceptable quality.
- [ ]  Test the website to ensure that the images load correctly and that performance is improved.

---
**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs).

### PR DESCRIPTION
**Pull request by Mandeep56Singh**
**Description**
This commit do below things:
- Convert all png to webP
- resolve path for converted png
- reduces size of members images

This PR fixes #5854

**Notes for Reviewers**
- ## The impact of commit can mainly be seen in through this video

### before: the content block due to having 3mb image to load
https://github.com/user-attachments/assets/06a44b60-7890-4c3e-a777-71b30ac199a7

### after: the image reduced to less than 100kb while maintaining good quality
https://github.com/user-attachments/assets/f36d6359-0b0d-4472-b919-5f01ea7182ff

- ## nearly 25% size reduction in images from 380 to 288mb 
 this calculation includes gif,webp,png,jpg,jpeg format
Screenshot (463)

I think the size of images can further reduced if dimension of imags are altered, the image above displayed video has unusaully high resolution 12k * 16k and size of 3mb. 
I only altered the dimension of this image to 1440 * 1920 size, resulting in  80kb size while having same quality.
rest of images are just optimized webp
  

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 2dca178f1535bd16c8a830415804b84e31ef4a52 at: https://66db4498d116e51aec1fefc4--layer5.netlify.app

**Comment by Ashparshp** [comment]
@Mandeep56Singh, Thank you for your contribution! Let's discuss this during the website call on Monday at 5:30 PM IST (7:00 AM CT). Please consider adding it as an agenda item to the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit?usp=drivesdk) || [meeting link](https://meet.layer5.io/websites).

**Comment by Ashparshp** [comment]
@Mandeep56Singh, I'm adding this as an agenda item to the meeting minutes. Please join the evening call.

**Comment by Mandeep56Singh** [comment]
@Ashparshp  is this pull request valid to get merged ? should go with  resolving the conflicts.

File: src/collections/integrations/_images/meshmap-visualizer.png
Comment (Ashparshp) [inline src/collections/integrations/_images/meshmap-visualizer.png:1]: Completely removed and not added?

**Comment by leecalcote** [comment]
Thanks for working on this, @Mandeep56Singh 👍

**Comment by leecalcote** [comment]
There are some merge conflicts to update.

**Comment by Ashparshp** [comment]
@Mandeep56Singh,
You've done an excellent job! I am still reviewing the images, and there are a few files left to review to ensure there are no issues. Please bear with me for a bit longer.

**Comment by sudhanshutech** [comment]
@Mandeep56Singh merge conflicts

**Comment by l5io** [comment]
🚀 Preview for commit d0ab98b0daed3bd4466e06f1cd6d02aea2847db8 at: https://66e916e793143aa0ca7dd280--layer5.netlify.app

**Comment by Mandeep56Singh** [comment]
thanks guys, for your feedbacks.
 I have made some changes kindly review it.

**Comment by leecalcote** [comment]
I think we're doing good here so far. Let's do a few spot checks to confirm. Here's one to reflect on:

Before - https://raw.githubusercontent.com/layer5io/layer5/b4df96153c3184643623e88cc744e67cbfdd81d1/src/collections/blog/2023/08-23-importance-of-github-stars/five-image.png

After - https://github.com/layer5io/layer5/blob/2dca178f1535bd16c8a830415804b84e31ef4a52/src/collections/blog/2023/08-23-importance-of-github-stars/five-image.webp

In Context (bottom of page) - https://layer5.io/blog/community/on-the-importance-of-github-stars

**Comment by Mandeep56Singh** [comment]
@leecalcote it's quite weird its perfectly working fine in [netlify build](https://66e916e793143aa0ca7dd280--layer5.netlify.app/blog/community/on-the-importance-of-github-stars)

Screenshot (474)

Am I missing something ?

**Comment by Mandeep56Singh** [comment]
@Ashparshp thanks.
About the suggestion, does this mean I have to the start from scratch  but part by part ?
it seems a good idea to me, But my pc isn't able to run layer5 project. so,I am using code spaces It has limited time usage and I am running out of it.
also we can take a quick  look at [netlify build](https://66e916e793143aa0ca7dd280--layer5.netlify.app/blog/community/on-the-importance-of-github-stars) for breaking changes.

**Comment by Ashparshp** [comment]
@Mandeep56Singh, are you a college student? If yes, please apply for the GitHub Student Developer Pack. It will help you run machines smoothly without limits issues. If not, no worries—I will review all these.

**Comment by lekaf974** [comment]
> @Ashparshp thanks. About the suggestion, does this mean I have to the start from scratch but part by part ?  

I think this would be a better approach since it will be easier for review and also reduce risk of issue

**Comment by Mandeep56Singh** [comment]
@Ashparshp  Ok, then please divide this into small task, and It would be way good if you also tell me the required image dimension for that task.
and let's get started.

**Comment by Ashparshp** [comment]
@Mandeep56Singh,

Please create PRs in smaller parts, with no more than 20-25 image changes at once. You can start with any of the tasks and we can merge them step by step.

**Comment by sudhanshutech** [comment]
@Mandeep56Singh i still not heard from you?

**Comment by sudhanshutech** [comment]
@Mandeep56Singh i saw you have another pr which you are doing in chunks so seems we can close this pr and you carry forward from there.

**Pull request closed without merging**